lib/lightning_web/endpoint.ex

```diff
@@ -54,7 +54,12 @@ defmodule LightningWeb.Endpoint do
   plug Plugs.WebhookAuth
 
   plug Plug.Parsers,
-    parsers: [:urlencoded, :multipart, :json],
+    parsers: [
+      :urlencoded,
+      :multipart,
+      # Increase to 10MB max request size only for JSON parser
+      {:json, length: 10_000_000}
```

Comment:
quick note here after a call with @jyeshe where we tested the various configuratio:

1. we need to keep the `:length` setting on line 61 here (but we should move it into runtime config so we can set via ENV)
2. in addition, we _should_ set a `:max_skip_body_length` so that we can stop attacks! the 10MB length limit reads the whole body into memory and responds with a very polite "hey, that's too big." setting `:max_skip_body_length` to something like `100MB` will simply cut the connection dead as soon as it detects more than 100MB and will be very useful to prevent DDoS attacks (or something like that)

@stuartc , this was an interesting call... we can't quite tell why `max_skip_body_length` isn't working in our implementation, but the `{:json, length: 10_000_000}` setting does the trick for now 🤷

Reply:
Sorry `protocol_options` should be under `http`: 
:http - the configuration for the HTTP server. It accepts all options as defined by [Plug.Cowboy](https://hexdocs.pm/plug_cowboy/). Defaults to false